cs/PerfTest/PerfTest.cs

```diff
@@ -0,0 +1,293 @@
+                        GC.Collect();
+                        Thread.Sleep(pauseMs);
+                    }
+
+                    if (testRun.TestResult.UseVarLenValue)
+                    {
+                        var fht = new FHT<VarLenValue, VarLenOutput, VarLenFunctions, UnusedSerializer<VarLenValue>>(
+                            false, testRun.TestResult.UseVarLenValue, testRun.TestResult.UseObjectValue, useReadCache: testRun.TestResult.UseReadCache);
+                        RunIteration(fht, testRun);
+
+                    }
+                    else if (testRun.TestResult.UseObjectValue)
+                    {
+                        var fht = new FHT<CacheObjectValue, CacheObjectOutput, CacheObjectFunctions, CacheObjectSerializer>(
+                            false, testRun.TestResult.UseVarLenValue, testRun.TestResult.UseObjectValue, useReadCache: testRun.TestResult.UseReadCache);
+                        RunIteration(fht, testRun);
+
+                    } else
+                    {
+                        switch (CacheGlobals.DataSize) {
+                            case 8:
+                                RunBlittableIteration<CacheBlittableValue8>(testRun);
+                                break;
+                            case 16:
+                                RunBlittableIteration<CacheBlittableValue16>(testRun);
+                                break;
+                            case 32:
+                                RunBlittableIteration<CacheBlittableValue32>(testRun);
+                                break;
+                            case 64:
+                                RunBlittableIteration<CacheBlittableValue64>(testRun);
+                                break;
+                            case 128:
+                                RunBlittableIteration<CacheBlittableValue128>(testRun);
+                                break;
+                            case 256:
+                                RunBlittableIteration<CacheBlittableValue256>(testRun);
+                                break;
+                            default:
+                                throw new InvalidOperationException($"Unexpected Blittable data size: {CacheGlobals.DataSize}");
+                        }
+                    }
+                }
+                testRun.Finish();
+                results.Add(testRun.TestResult);
+            }
+
+            sw.Stop();
+
+            if (results.Results.Length == 0)
+            {
+                Console.WriteLine("No tests were run");
+                return;
+            }
+
+            Console.WriteLine($"Completed {results.Results.Length} test run(s) in {TimeSpan.FromMilliseconds(sw.ElapsedMilliseconds)}");
+            if (!string.IsNullOrEmpty(resultsFilename))
+            {
+                results.Write(resultsFilename);
+                Console.WriteLine($"Results written to {resultsFilename}");
```

Comment:
How does performance compare to FASTER.benchmark when settings are identical? So we have a reference for the performance behavior.